### PR TITLE
OSX build

### PR DESCRIPTION
**Pull request by dfornika**
Addresses #3.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,14 +13,15 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  skip: True  # [osx or win]
-  number: 0
```
Comment (SylvainCorlay) [inline recipe/meta.yaml]: I don't think that you need to bump the build number if this does not change the outcome of the linux build.

Comment (dfornika) [reply recipe/meta.yaml]: Ok, thanks. I wasn't sure about that.

**Comment by dfornika** [comment]
I don't think the OS X build is being tested in a CI environment. I'll have to look into how to set that up.

**Comment by jakirkham** [comment]
So to enable another OS that was previously disabled, one needs to change the `skip` and [re-render]( https://github.com/conda-forge/conda-smithy/blob/master/README.md#re-rendering-an-existing-feedstock ) the feedstock. Would make sure you have the latest `conda-smithy` and `conda-build-all` before re-rendering. Installation is also explained in the same README.

Also would add that Travis CI (used for macOS builds) is working through some issues. So the build might take a while to complete. That said, would recommend going ahead and doing the work to get that build queued.

**Pull request closed without merging**